_includes/feed.liquid

```diff
@@ -11,6 +11,9 @@ section: smartContract
     <p>For LINK token and Faucet details, see the <a href="../link-token-contracts/">LINK Token Contracts</a> page.</p>
 {% endif %}
 
+<p>Your applications are only as reliable as the data that they use</p>
+<rdme-callout theme="warn" icon="🚧" title="Your applications are only as reliable as the data that they use" value="When you run a production application, be aware of the risks associated with each data provider that you use."><p>Your applications are only as reliable as the data that they use</p><p>When you run a production application, be aware of the risks associated with each data provider that you use. See the <a href="../using-chainlink-reference-contracts/#use-data-feeds-with-reliable-data-sources">Introduction to Data Feeds</a> page to learn how to ensure that your data feeds use reliable data sources.</p></rdme-callout>
```

Comment:
Do you want to swap this for markdown instead? I _think_ you can use <markdown> to inject markdown anywhere. I haven't tested this in a liquid file.

Reply:
MD didn't work on my first attempt. I'll experiment some more. We can update it in a followup PR.